R/attribute-element.R

```diff
@@ -357,16 +357,30 @@ add_interval_ratio <- function(type = c("interval", "ratio"), units, unit_precis
                                      number_type = "Please provide what type of numbers are being used.", 
                                      minimum = "Please provide a minimum theoretical value if applicable.",
                                      maximum = "Please provide a maximum theoretical value if applicable.")
-    if (is.null(type) | is.null(units) | is.null(unit_precision) | is.null(number_type)) {
+    if (is.null(type) | is.null(units) | is.null(number_type)) {
       stop(interval_error_message, call. = FALSE)
     } 
   } 
   type <- match.arg(type)
   
-  interval_ratio <- list(unit = list(standardUnit = units),
-                         precision = unit_precision,
-                         numericDomain =
-                           list(numberType = number_type))
+  if (units %in% standard_units) {
```

Comment:
should `standard_units` have the package namespace `EMLaide::standard_units`? i like that style better, lets me know where something is coming from

Reply:
Agreed! Changed to EMLaide::standard_units